modules/router/src/test/autoRebalance.spec.ts

```diff
@@ -119,6 +119,7 @@ describe(testName, () => {
       });
 
       const hash = mkBytes32("0xa");
+      // Based on the numbers hard coded below, the auto rebalancer should get 37.5 (half the difference).
```

Comment:
Moar tests pls -- ideally around queueing. Make sure to also test this locally

Reply:
Yep, will do